examples/kubernetes/pomerium-certificates.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: pomerium-cert
+  namespace: pomerium
+spec:
+  secretName: pomerium-tls
+  issuerRef:
+    name: pomerium-issuer
+    kind: Issuer
+  usages:
+    - server auth
+    - client auth
+  dnsNames:
+    - pomerium-proxy.pomerium.svc.cluster.local
+    - pomerium-authorize.pomerium.svc.cluster.local
+    - pomerium-databroker.pomerium.svc.cluster.local
+    - pomerium-authenticate.pomerium.svc.cluster.local
+    - "*.localhost.pomerium.io" # Quotes are required to escape the wildcard
```

Comment:
Maybe a comment to replace with your own domain.  Should we mark changes that need to be made per their environment with a `# TODO` block?

Reply:
+1 on adding a todo note.

Reply:
That note is made in the file referencing this script ([link](https://github.com/pomerium/pomerium/pull/2390/files/c3e4e297943ee95c56d85d7b538ac85cf6ce915c#diff-679005b27825031da5927255d27a7e4c5b96c0f798120ad3ccf7131939847526R129-R131).

Do you want an inline comment to replace that note, or in addition to?

Reply:
In addition.  I'm thinking of folks having downloaded and checked it out - often their editors will identify those comments.